### DISCUSSION TITLE
Wrong lua commenting code-style

### DISCUSSION THREAD

**Discussion by Hollow111** [tarantool/doc#2809] [General]
["Commenting" topic](https://tarantool.io/en/doc/2.0/dev_guide/lua_style_guide) in Lua style guide.

Functions commenting should be like following:
```
--
-- Connect to a remote server, do handshake.
-- @param host Hostname.
-- @param port TCP port.
-- @param timeout Timeout to connect and receive greeting.
--
-- @retval nil, err Error occured. The reason is returned.
-- @retval two non-nils A connected socket and a decoded greeting.
--
local function establish_connection(host, port, timeout)
```

**Comment by Totktonada** [comment]
It is topic for discussion. Say, we can use ldoc syntax and gain ability to generate some api documentation.

**Comment by lenkis** [comment]
Please discuss this with @kyukhin. Prescribing ldoc as a standard is just a minor change for the guidelines, but a tangible twist of our Lua dev practice.

**Comment by Totktonada** [comment]
See also http://stevedonovan.github.io/ldoc/manual/doc.md.html#Extension_modules_written_in_C
